### ISSUE TITLE
Fcitx5 input method is not working in FeelUOwn

### ISSUE THREAD

**Issue by BruceZhang1993** [cosven/FeelUOwn#235]
- OS: Arch Linux
- FeelUOwn: git master latest
- Fcitx Version: r405.9e15086-1

With **fcitx5-qt5-git** installed.

Env:
```
GTK_IM_MODULE=fcitx5
QT_IM_MODULE=fcitx5
XMODIFIERS=@im=fcitx
```

**Comment by chen-chao** [comment]
What desktop environment do you use? My fcitx works well in KDE.

**Comment by BruceZhang1993** [comment]
@chen-chao KDE + fcitx5   that is a strange problem.

**Comment by chen-chao** [comment]
My `fcitx --version` yields 4.2.9.5. Maybe the prolem is caused by fcitx5 and is irrelevant to feeluown.

**Comment by BruceZhang1993** [comment]
@chen-chao I can use fcitx5 in other pyqt5 apps normally, so it must be something wrong. - - Keep digging~

**Comment by BruceZhang1993** [comment]
Latest fcitx5 and related packages has been released, and this issue seems to be solved on FeelUOwn 2.2. 
So closing now.
